docs/config/_default/hugo.toml

```diff
@@ -61,10 +61,83 @@ weight = 1
 
 [[menu.main]]
     name = "GitHub"
-    weight = 50
+    weight = 60
+    url = "https://github.com/cilium/tetragon/"
+    pre = "<i class=\"fa-brands fa-github\"></i>"
+
+[[menu.navbar_main]]
+    name = "Documentation"
+    weight = 10
+    url = "/docs"
+
+[[menu.navbar_main]]
+    name = "Community"
+    weight = 20
+    url = "https://cilium.io/get-involved/"
+
+[[menu.navbar_main]]
```

Comment:
as a follow up because let's merge this PR, it's been hanging for a while: maybe we should remove the `menu.main`, they are useless with your current implementation of the navbar no? Is it used somewhere else?

And maybe I would name it something like `menu.navbar_custom` or whatever to show that it's using our custom drop down implem?

Reply:
<img width="1278" alt="image" src="https://github.com/user-attachments/assets/3193b744-4b20-4025-975f-0f168d6c3573" />

We’re still using `menu.main` in the footer (and we can probably rename it to reflect that better)

The one in the navbar can be renamed without a problem